workbench-core/swb-common-ui/.eslintrc.js

```diff
@@ -13,5 +18,13 @@ module.exports = {
     'import/no-named-as-default-member': 0,
     // disabling this rule because next expects a pages directory which we dont have here
     '@next/next/no-html-link-for-pages': 0
+  },
+  settings: {
+    react: {
+      version: 'detect'
+    }
   }
 };
+
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
+require('@rushstack/eslint-config/patch/modern-module-resolution');
```

Comment:
why is this at the top and bottom of the file?

Reply:
Good catch Thanks !